install/kubernetes/cilium/values.yaml

```diff
@@ -119,10 +119,18 @@ global:
   # disabled.
   installIptablesRules: true
 
-  # masquerade enables masquerading of traffic leaving the ndoe for
+  # masquerade enables masquerading of traffic leaving the node for
   # destinations outside of the cluster.
   masquerade: true
 
+  # bpfMasquerade enables masquerading with BPF instead of iptables
+  bpfMasquerade: false
```

Comment:
Why aren't we enabling this by default for new instalations?

Reply:
Once we have enough confidence about this feature, I'm happy to enable this by default;-)

Reply:
@brb so when will we have confidence about it? Does this mean the feature is in beta? If it's not enabled for new installations will anyone use it?

Reply:
Let's discuss it during the next meeting.